README.md

```diff
@@ -172,7 +172,7 @@ Other than this, the NODE_NAME environment variable must be set where the CSI sn
 
 * `--retry-interval-start`: Initial retry interval of failed volume snapshot creation or deletion. It doubles with each failure, up to retry-interval-max. Default value is 1 second.
 
-* `--retry-interval-max`: Maximum retry interval of failed volume snapshot creation or deletion. Default value is 5 minutes.
+* `--retry-interval-max`: Maximum time to wait for CRDs to appear. Default is 10 seconds.
```

Comment:
This retry-interval-max is also for the failed volume snapshot creation/deletion retry interval, not just for controller startup. 10 seconds is too short for those operations.

Reply:
Sorry, I had not modified the correct part of the README, it should e better now :sweat_smile: